Queue.java

```diff
@@ -5,15 +5,20 @@
 import java.util.*;
 
 class Queue {
+    private Stack<Integer> stack1;
+    private Stack<Integer> stack2; 
 
 
     public Queue() {
+        stack1 = new Stack<Integer>();
+        stack2 = new Stack<Integer>();
 
     }
 
     // Push element x to the back of queue.
+    // time complexity: O(1), space complexity:O(N)
     public void enqueue(int x) {
-
+        stack1.push(x);
```

Comment:
The idea is to push the element to the queue, but line 21st says the element is moved to the stack. When you push an element to stack, how can it behave as an enqueue in the queue?